src/Models/Classes/AdapterEntityCache.ts

```diff
@@ -18,6 +18,11 @@ class AdapterEntityCache<T extends IAdapterData> {
         this.maxAgeMs = maxAgeMs;
     }
 
+    setMaxAgeMs(maxAge: number): void {
+        this.maxAgeMs = maxAge;
+        this.cachedEntities = new Map();
```

Comment:
so does this mean whenever we change the polling interval we will lose the cachedEntities, this is correct, right?